code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -67,10 +69,26 @@
 	flags =  CONDUCT
 	slot_flags = SLOT_BACK
 	origin_tech = "combat=4;materials=3;powerstorage=3"
-	ammo_type = list(/obj/item/ammo_casing/energy/laser/heavy)
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/accelerator)
 	pin = null
 	ammo_x_offset = 3
 
+/obj/item/ammo_casing/energy/laser/accelerator
+	projectile_type = /obj/item/projectile/beam/laser/accelerator
+	select_name = "accelerator"
+	fire_sound = 'sound/weapons/lasercannonfire.ogg'
+
+/obj/item/projectile/beam/laser/accelerator
+	name = "accelerator laser"
+	icon_state = "scatterlaser"

```

Comment:
I just tested with the scatter one though, it looks good having a little circle grow gigantic as it flies